mne/preprocessing/xdawn.py

```diff
@@ -111,7 +107,8 @@ def _construct_signal_from_epochs(epochs):
     return data
 
 
-def least_square_evoked(epochs, return_toeplitz=False):
+def least_square_evoked(epochs_data, events, event_id, tmin, tmax, sfreq,
```

Comment:
IMO the function should have been private in the first place.  

@kaichogami I would keep and deprecate the public function, and make a private function that follow yours. ok @dengemann, @agramfort ?